common/FileUtils.java

```diff
@@ -183,7 +186,7 @@ public String getCurrentWorkingDirectoryName() {
             fileSystem.getPath(expandHomeDirectory(path)),
             100000, // Folder depth
             (unused, unused2) -> true)) {
-      return ImmutableList.sortedCopyOf(paths.map(Path::toString).collect(Collectors.toList()));
+      return ImmutableList.sortedCopyOf(paths.map(Path::toString).collect(toList()));
```

Comment:
Let's keep `Collectors` - that way it's clear it's not a local method.

Reply:
In my personal opinion, `collect(toList())` reads better because of how it looks like prose. (I'm also reasonably sure that's how the OpenJDK devs intended `Collectors` to be used.) But I won't insist. :)

Reply:
Hmm, actually, I realise that the result of `collect(toList())` is being passed to a call to `ImmutableList.sortedCopyOf`.

Thus I wonder if the following would read better and be equally performant:

```java
return paths.map(Path::toString).sorted().collect(toImmutableList());
```

What do you think?

Reply:
Valerii, can you check the shorter version? It looks nice.
About removing the `Collectors.`, I think that the convention of methods not bring imported has the benefit that when you see a method, you don't need to think twice to know that it's local.
I can see the benefits of shortening the code, but once you do it here (and maybe in a few other places here and there), you lose that confidence across the whole codebase.

Reply:
I tried to use the shorter version:
 ```
return paths.map(Path::toString).sorted().collect(toImmutableList());
```
`toImmutableList()` required additional import:
```
import static autovalue.shaded.com.google$.common.collect.$ImmutableList.toImmutableList;
```
After these changes, `listContentsRecursively()` method will return `autovalue.shaded.com.google$.common.collect.$ImmutableList` instead `com.google.common.collect.ImmutableList`. 

And it requires changing all the code where `listContentsRecursively()` method is used.